src/main.cpp

```diff
@@ -64,8 +64,49 @@ static void servoSetUs(unsigned int servo, int usec)
     g_OutputsUs[servo] = usec;
 }
 
+
+static void outputFailsafeValues()
+ {
+    for (unsigned int out=0; out<NUM_OUTPUTS; ++out)
+        {
+            if (OUTPUT_FAILSAFE[out] == fsaNoPulses)
+                servoSetUs(out, 0);
+            else if (OUTPUT_FAILSAFE[out] != fsaHold)
+                servoSetUs(out, OUTPUT_FAILSAFE[out]);
+            // else fsaHold does nothing, keep the same value
+        }
+}
+
+
+#if defined(ARMSWITCH)
+    short armCount = 0;
+#endif
+
 static void packetChannels()
 {
+    #if defined(ARMSWITCH)
+        // If channel 5 doesn't send "arm" signal (value 2000), output failsafe only
+        if (crsf.getChannel(5) != 2000)

```

Comment:
Standard ELRS defines armed as > 1500us, not != 2000

Reply:
This is incorrect, on the ELRS webpage, there are two big warnings, stating that it should be 2000: https://www.expresslrs.org/software/switch-config/